app/presenters/publishing_api/organisation_presenter.rb

```diff
@@ -81,14 +109,198 @@ def image
       }
     end
 
-    def brand
-      brand_colour = item.organisation_brand_colour
-      brand_colour ? brand_colour.class_name : nil
+    def foi_exempt
+      item.foi_exempt
+    end
+
+    def corporate_information_pages
+      item.published_corporate_information_pages.map do |cip|
+        {
+          title: cip.title,
+          href: Whitehall.url_maker.public_document_path(cip)
+        }
+      end
+    end
+
+    def featured_links
+      item.featured_links.map do |link|
+        {
+          title: link.title,
+          href: link.url
+        }
+      end
+    end
+
+    def featured_documents
+      item.feature_list_for_locale(I18n.locale).current.map do |feature|
+        if feature.document
+          featured_documents_editioned(feature)
+        elsif feature.topical_event
+          featured_documents_topical_event(feature)
+        elsif feature.offsite_link
+          featured_documents_offsite_link(feature)
+        end
+      end
+    end
+
+    def featured_documents_editioned(feature)
+      # Editioned formats (like news) that have been featured
+      edition = feature.document.published_edition
+      {
+        title: edition.title,
+        href: Whitehall.url_maker.public_document_path(edition),
+        image: {
+          url: feature.image.url,
+          alt_text: feature.alt_text
+        },
+        summary: edition.summary,
+        public_updated_at: edition.public_timestamp,
+        document_type: edition.display_type
+      }
+    end
+
+    def featured_documents_topical_event(feature)
+      # Topical events that have been featured
+      topical_event = feature.topical_event
+      {
+        title: topical_event.name,
+        href: Whitehall.url_maker.polymorphic_path(topical_event),
+        image: {
+          url: feature.image.url,
+          alt_text: feature.alt_text
+        },
+        summary: topical_event.description,
+        public_updated_at: topical_event.start_date,
+        document_type: nil # We don't want a type for topical events
+      }
+    end
+
+    def featured_documents_offsite_link(feature)
+      # Offsite links that have been featured
+      offsite_link = feature.offsite_link
+      {
+        title: offsite_link.title,
+        href: offsite_link.url,
+        image: {
+          url: feature.image.url,
+          alt_text: feature.alt_text
+        },
+        summary: offsite_link.summary,
+        public_updated_at: nil, # We don't want a date for offsite links
+        document_type: offsite_link.humanized_link_type
+      }
+    end
+
+    def ministers
+      item.ministerial_roles.map do |role|
```

Comment:
All the roles are ordered by their presenters, so 👍

Reply:
I've refactored this to reduce duplication and also ensure the ordering is correct.